### ISSUE TITLE
incompatible encoding regexp match (UTF-8 regexp with ASCII-8BIT string)

### ISSUE THREAD

**Issue by pedro380085** [fastlane/fastlane#8600]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

[15:52:55]: Starting login with user 'xxxx'
[15:52:59]: Successfully logged in
[15:52:59]: Fetching profiles...
+------------------+------------+
|         Lane Context          |
+------------------+------------+
| PLATFORM_NAME    |            |
| LANE_NAME        | demo       |
| PRODUCE_APPLE_ID | |
+------------------+------------+
[15:53:00]: incompatible encoding regexp match (UTF-8 regexp with ASCII-8BIT string)

I recently started getting this error when running sigh. Anybody knows what it might be?

**Comment by pedro380085** [comment]
Error line:

/usr/local/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:91:in `scan': [!] incompatible encoding regexp match (UTF-8 regexp with ASCII-8BIT string) (Faraday::ParsingError)

**Comment by Eriick** [comment]
I have the same problem after to update match.! :|

/Users/admin/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/plist-3.1.0/lib/plist/parser.rb:91:in `scan': [!] incompatible encoding regexp match (UTF-8 regexp with ASCII-8BIT string) (Faraday::ParsingError)

**Comment by pedro380085** [comment]
It seems like Apple updated some on their backend, because sigh fails when processing the response.

**Comment by KrauseFx** [comment]
Everything seems to be working just fine on my machine, can you share some more background that can help us reproduce the issue? Did you upgrade your Ruby/Xcode recently?

**Comment by pedro380085** [comment]
Here is the complete stack trace:

```
	from /usr/local/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:91:in `parse'
	from /usr/local/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:29:in `parse_xml'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/helper/plist_middleware.rb:10:in `block in <class:PlistMiddleware>'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:50:in `parse'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:39:in `process_response'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:32:in `block in call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday-0.11.0/lib/faraday/response.rb:61:in `on_complete'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.11.0.1/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday-0.11.0/lib/faraday/rack_builder.rb:139:in `build_response'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday-0.11.0/lib/faraday/connection.rb:377:in `run_request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/faraday-0.11.0/lib/faraday/connection.rb:177:in `post'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/client.rb:546:in `block in send_request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/client.rb:392:in `with_retry'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/client.rb:545:in `send_request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/client.rb:436:in `request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/portal_client.rb:474:in `provisioning_profiles_via_xcode_api'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/provisioning_profile.rb:271:in `all'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/provisioning_profile.rb:320:in `find_by_bundle_id'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/sigh/lib/sigh/runner.rb:64:in `fetch_profiles'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/sigh/lib/sigh/runner.rb:20:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/sigh/lib/sigh/manager.rb:7:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/actions/sigh.rb:18:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:92:in `block in parsing_binding'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:105:in `block (2 levels) in run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:153:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/runner.rb:446:in `run_active_command'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:304:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/bin/fastlane:22:in `<main>'
```

**Comment by KrauseFx** [comment]
```
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/portal_client.rb:474:in `provisioning_profiles_via_xcode_api'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/provisioning_profile.rb:271:in `all'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/spaceship/lib/spaceship/portal/provisioning_profile.rb:320:in `find_by_bundle_id'
```

Interesting, why do you access `provisioning_profiles_via_xcode_api`, can you try to use the other API? Did you set `SPACESHIP_AVOID_XCODE_API`?

**Comment by pedro380085** [comment]
Hm, no. My sigh call is very simple, only parameter is `skip_certificate_verification: false`.

**Comment by pedro380085** [comment]
If I use `ENV["SPACESHIP_AVOID_XCODE_API"] = "true"`, it timeouts:

```
from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:261:in `select_with_timeout'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:261:in `select_with_timeout'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:131:in `rescue in block in connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:131:in `rescue in block in connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:112:in `block in connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:112:in `block in connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `each'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `each'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:153:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:29:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:9:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `new'
: Interrupt
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `socket'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:100:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/mock.rb:48:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/instrumentor.rb:26:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:249:in `request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon.rb:239:in `post'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/update_checker/update_checker.rb:136:in `finished_running'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/update_checker/update_checker.rb:39:in `block in show_update_status'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/update_checker/update_checker.rb:37:in `fork'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/update_checker/update_checker.rb:37:in `show_update_status'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/cli_tools_distributor.rb:68:in `take_off'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:261:in `select_with_timeout'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:131:in `rescue in block in connect'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:112:in `block in connect'
	from /usr/local/bin/fastlane:22:in `<main>'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `each'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:153:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:29:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:9:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `socket'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:100:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/mock.rb:48:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/instrumentor.rb:26:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:249:in `request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon.rb:239:in `post'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:81:in `block in did_finish'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:79:in `fork'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:79:in `did_finish'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/runner.rb:290:in `did_finish'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/lane_manager.rb:75:in `finish_fastlane'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/lane_manager.rb:68:in `cruise_lane'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:105:in `block (2 levels) in run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:153:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/runner.rb:446:in `run_active_command'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:304:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/bin/fastlane:22:in `<main>'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:103:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:153:in `connect'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/socket.rb:29:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/ssl_socket.rb:9:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:403:in `socket'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:100:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/mock.rb:48:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/instrumentor.rb:26:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/middlewares/base.rb:16:in `request_call'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon/connection.rb:249:in `request'
	from /usr/local/lib/ruby/gems/2.3.0/gems/excon-0.55.0/lib/excon.rb:239:in `post'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:81:in `block in did_finish'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:79:in `fork'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/tool_collector.rb:79:in `did_finish'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:81:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:304:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'
	from /usr/local/lib/ruby/gems/2.3.0/gems/fastlane-2.22.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/bin/fastlane:22:in `<main>'
```

**Comment by koutalou** [comment]
@pedro380085
I resolved the problem by #8611.

Maybe you can temporarily resolve by adding follows code,

```
## **your gem directory**/gems/plist-3.2.0/lib/plist/parser.rb:90
      @xml = @xml.force_encoding("UTF-8") # Add this code
      @scanner = StringScanner.new( @xml )
```

or  https://github.com/fastlane/fastlane/pull/8611/files#diff-2628d3e9348dc001ed91a9749a936d18R10

**Comment by biolinh** [comment]
@koutalou I do as you said but it returns  error:

````
/Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:91:in `scan': [31m[!] incompatible encoding regexp match (UTF-8 regexp with ASCII-8BIT string)[0m (Faraday::ParsingError)
	from /Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:91:in `parse'
	from /Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/plist-3.2.0/lib/plist/parser.rb:29:in `parse_xml'
	from /Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/fastlane-2.15.1/spaceship/lib/spaceship/helper/plist_middleware.rb:10:in `block in <class:PlistMiddleware>'
	from /Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/faraday_middleware-0.10.1/lib/faraday_middleware/response_middleware.rb:48:in `parse'
	from /Users/jenkins/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/faraday_middleware-
```

**Comment by pedro380085** [comment]
It works for me, thanks @koutalou!

**Comment by Eriick** [comment]
Too it works for me, thks @koutalou

**Comment by nafu** [comment]
This is resolved at [_fastlane_ 2.23.0](https://github.com/fastlane/fastlane/releases/tag/2.23.0) :rocket: So, I'll close this.

Thanks @koutalou for the PR 🙂
Thanks for confirming @pedro380085 @Eriick 👍

**Comment by crystalmind** [comment]
EOF
